packages/input-select/src/Select.js

```diff
@@ -119,7 +123,20 @@ class Select extends React.Component {
         innerRef={ref}
         refName={refName}
         image={image}
-      />
+      >
+        {multi &&
+          items.map(
+            item =>
+              this.state.selectedValue.includes(item.value) && (
+                <Tag
+                  key={item.value}
+                  value={item.value}
+                  text={item.label}
+                  onRemove={this.onRemoveTag}
```

Comment:
We are getting this error when an item is selected:
```
Warning: Unknown event handler property `onRemove`. It will be ignored.
    in div (created by Context.Consumer)
    in StyledComponent (created by styled.div)
    in styled.div (created by Tag)
```

The fix is pretty simple, would you mind adding the fix to this pr?
https://github.com/CraveFood/farmblocks/blob/master/packages/tag/src/components/Tag.js#L9-L11
Instead of passing props, we could pass only the remaining props to `StyledTag`

Reply:
I did try that. The problem is that `StyledTag` needs the `onRemove` prop:
https://github.com/CraveFood/farmblocks/blob/6f28777edc7ce91370820da2493b58bcb925fd68/packages/tag/src/components/styledComponent/StyledTag.js#L6

Reply:
Ouch lol

Reply:
What about 

```jsx
<StyledTag rightPadding={onRemove? "0px" : "8px"} {...rest}>
```

Reply:
Oh, the warning was just because it was an event handler name. Converting it to a boolean solves the problem.